Jucer2CMake/CMakeLists.txt

```diff
@@ -71,7 +66,10 @@ target_compile_definitions(Jucer2CMake_JUCE PUBLIC
 )
 
 if(APPLE)
-  target_compile_options(Jucer2CMake_JUCE PRIVATE -Wno-deprecated-declarations)
+  target_compile_options(Jucer2CMake_JUCE PRIVATE
+    -Wno-deprecated-declarations
+    -Wno-register
```

Comment:
What is -Wno-register with C++17 ? a new unnecessary warning generated by setting cxx17 ?

Reply:
When building older versions of JUCE with C++17 we get:
```
error: ISO C++17 does not allow 'register' storage class specifier [-Wregister]
```
from third-party libraries included in JUCE (jpglib and zlib). There's nothing much we can do but disable that warning.